### PR TITLE
Added possum@1.0.4 [author] [new]

### PR DESCRIPTION
**Pull request by mnichols**
**Origin of files** cdnjs-importer pulled down from npm and `npm run build`. 
You can download the source at https://github.com/mnichols/possum/releases/tag/v1.0.4

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): mnichols/possum
- Official website (optional, not the repository): http://mnichols.github.io/possum/
- NPM package url (optional): https://www.npmjs.com/package/possum
- GitHub / Bitbucket popularity (required):
  - Count of stars: 6
  - Count of watchers: 2
  - Count of forks: 2
- NPM download stats (optional):
  - Downloads in the last day: 4
  - Downloads in the last week: 421
  - Downloads in the last month: 939
# Essential checklist
- [x] I'm the author of this library
  - [x] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [ ] Auto-update setup
- [ ] Auto-update target/source is valid.
- [ ] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/possum/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "possum",
+  "version": "1.0.4",
+  "description": "I am able. A State Machine.",
+  "babel": {
+    "presets": [
+      "es2015"
+    ]
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/possum/package.json]: Please remove "`babel`".

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR as your practice, thanks.

File: ajax/libs/possum/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "cuid": "^1.2.4",
+    "stampit": "^2.1.0"
+  },
+  "npmName": "possum",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
```
Comment (x09326) [inline ajax/libs/possum/package.json]: The basepath should be ```src``` not ```/src```.
And ```src``` is not a good basepath.
You can modify it to empty and modify the files to ```src/*```.

If you modify these fields, the filename should be modified too.

**Comment by PeterDaveHello** [comment]
Hello, any updates here? Thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
Hi @mnichols , would you fix this PR and let it be merged ASAP?
If you are busy recently, I glad to help you to finish this PR.
Thank you.

**Comment by mnichols** [comment]
hi @sufuf3 I'm super tied up atm . Can i review next week or should I just close this and reopen when I can devote time to it?

**Comment by PeterDaveHello** [comment]
@mnichols No worriew, let's continue this next week :)

**Comment by sufuf3** [comment]
It's OK, so glad to hear you'd like to work on this PR. Let's work on next week.

**Comment by PeterDaveHello** [comment]
PS: If there are only few minor changes, I think we can also help you directly fix it and merge the PR, let me know what do you think :)

**Comment by sufuf3** [comment]
Hi @mnichols, If you'd like to work on this PR recently.
I think this PR need to be closed and send a new PR by creating a new branch.

Because this PR is a little bit old, and also we do not accept PR from master branch anymore. 
Many thanks!

**Comment by sufuf3** [comment]
@PeterDaveHello could we close this PR because there is no branch from fork repo?
And I don't know that is it good to let us to open an issue to add this lib?
Although there are 601 downloads in the last month and it didn't have enough popularity in current, it's fit in the past.

Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 cool. @mnichols feel free to send a new PR.

**Pull request closed without merging**